src/components/VTable/VTable.vue

```diff
@@ -75,7 +75,7 @@
         </thead>
         <tbody>
           <slot v-bind="{ items: cItems, ...$data, perPage }">
-            <tr v-for="(item, index) in cItems" :key="item.id" @click="$emit('click:row', item)">
+            <tr tabindex="0" v-for="(item, index) in cItems" :key="item.id" @click="$emit('click:row', item.original)">
```

Comment:
Im not sure adding a tabindex to the row is the best idea. I know it provides access to that row click event, but I have not seen anything that suggests table rows should be tabable from an a11y perspective. The last thing I want to do is make every table annoying to navigate. If there's 100 rows, that's 100 tabs just to get past it.

Reply:
That makes sense. I just found it odd that you could tab to the table container, but not inside it. I can take that out if you want.

How do you feel about the other change (return the original object instead of the item object)?